### ISSUE TITLE
Navigator profile list blank

### ISSUE THREAD

**Issue by tjbergst** [Teradata/jupyterextensions#1]
I did a manual install on a Windows 10 machine. I am able to establish a connection and query within a notebook just fine, however when I launch the navigator there are no profiles available in the dropdown. The browser console shows:
`Failed to load resource: the server responded with a status of 401 (Unauthorized)` against `:10002/profiles/profilesall:1`
When I try to navigate to the noted profiles url `http://localhost:10002/profiles/profilesall` I seem the same, request not authorized.

Is there somewhere else I should be adding connection profile information?

**Comment by ltammariello** [comment]
No, all connections are managed through the notebooks.  After you add your connection, be sure to hit the refresh button in the Navigator view.  Also, please don't specify any flags (e.g. --notebook-dir ) when launching jupyter lab.
